browser/admin/src/integrator/Xcu.ts

```diff
@@ -293,7 +306,7 @@ class Xcu {
 					const fieldset = document.createElement('fieldset');
 					fieldset.classList.add('xcu-settings-fieldset');
 					const legend = document.createElement('legend');
-					legend.textContent = key;
+					legend.textContent = _(key);
```

Comment:
key is code, isn't it (not for translation)?

Reply:
yes